x/dex/keeper/limit_order_tranche.go

```diff
@@ -57,6 +57,7 @@ func (k Keeper) SaveTranche(ctx sdk.Context, tranche *types.LimitOrderTranche) {
 	} else {
 		k.SetInactiveLimitOrderTranche(ctx, tranche)
 		k.RemoveLimitOrderTranche(ctx, tranche.Key)
+		ctx.EventManager().EmitEvents(getEventsDecTotalTickLiquidities())
```

Comment:
Either need to unify TickLiquidities, TotalLimitOrders TotalExpiringLimitOrders or have dec events for LimitOrders and ExpiringLimitOrders here as well